### DISCUSSION TITLE
closing a room without entering it

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3757] [Hubs Cloud Questions] [answered]
Hello, 
All is in the title ? I am searching for a solution to close a room without entering on it , maybe close it from the admin panel ? Have you some advice  ? 

Thx 
Marc

**Comment by robinkwilson** [comment, marked as answer]
Unfortunately you currently have to enter the room to close the room. You can't close a room from the admin panel yet, though you will be able to with the new Hubs API in the future. The new Hubs API will allow you to set `entry_mode` to `deny` to close the room. 

The API will be merged into Hubs Cloud in the next few releases (no date set yet). We're in the process of testing the API now.
